cmd/agent/app/integrations_test.go

```diff
@@ -142,3 +142,35 @@ func TestNormalizePackageName(t *testing.T) {
 	assert.Equal(t, normalizePackageName("datadog-checks_base"), "datadog-checks-base")
 	assert.Equal(t, normalizePackageName("datadog_checks_downloader"), "datadog-checks-downloader")
 }
+
+func TestParseWheelPackageNameValidCases(t *testing.T) {
+	tests := map[string]struct {
+		wheelFileName string
+		expectedName  string
+	}{
+		"name as first line":  {"datadog_cisco_aci_name_first_line_valid.whl", "datadog-cisco-aci-1"},
+		"name as second line": {"datadog_cisco_aci_name_second_line_valid.whl", "datadog-cisco-aci-2"},
+	}
+	for name, test := range tests {
+		t.Logf("Running test %s", name)
+		name, err := parseWheelPackageName("./testdata/integrations/" + test.wheelFileName)
```

Comment:
Maybe we should download to a temp dir instead?

Reply:
Mostly to not impact repo size, feel free to disregard if you think it's not worth it!

Reply:
Yes, that's a quite a valid concern. Yes, though about that and I already removed the python files from the wheel, the wheels contain only few metadata files. Each `.whl` is only about 5kb, for comparison the `integrations.go` is 23kb.

I would favour having the testdata in the repo, downloading the files from elsewhere might make the test more flaky, test might fail because the files are not available for some reasons.

Reply:
Remove all files except `METADATA` from the wheels, 3kb each now.